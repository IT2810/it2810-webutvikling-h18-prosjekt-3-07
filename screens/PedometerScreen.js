
import React from 'react';
import { View, StyleSheet, Text } from 'react-native';
import PedometerSensor from '../components/PedometerSensor'


class PedometerScreen extends React.Component {
  static navigationOptions= {
    title: 'Pedometer',
  };
  render() {

    return (
      <View style={styles.container}>
        <PedometerSensor />
      </View>
    );
  }
}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
    backgroundColor: '#fff',
  }
});
export default PedometerScreen;