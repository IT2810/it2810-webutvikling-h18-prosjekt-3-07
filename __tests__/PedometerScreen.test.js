import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import PedometerScreen from "../screens/PedometerScreen";

it("matches snapshot", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<PedometerScreen />);
  expect(result).toMatchSnapshot();
});
