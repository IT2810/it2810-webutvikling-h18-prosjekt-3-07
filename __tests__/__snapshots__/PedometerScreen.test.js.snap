// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <PedometerSensor
    setSteps={[Function]}
  />
  <PedometerProgressBar
    steps={0}
  />
</Component>
`;
